ajax/libs/vega-lite/package.json

```diff
@@ -16,6 +16,12 @@
       "files": [
         "vega-lite*.+(js|css|map)"
       ]
+    },
+    {
+      "basePath": "build",
+      "files": [
+        "**/vega-lite*.+(js|css|map)"
```

Comment:
We build every single `.ts` file and put it in `build`. I don't think they should be on a cdn.